docs/04-dacs.md

```diff
@@ -0,0 +1,69 @@
+# DAC Configuration
+
+While most Raspberry Pi boards come with an onboard sound card it is well known that the quality you can get from them is not the greatest. The Pi Foundation makes an excelent job at keeping the cost of their bords down, but that comes with some compromises, audio being one of them: the [audio circuitry](https://hackaday.com/2018/07/13/behind-the-pin-how-the-raspberry-pi-gets-its-audio/) does an OK job but it's nothing stellar.
```

Comment:
Fixes and recommendation:

"While most Raspberry Pi boards come with an onboard sound card, it's well known that the quality you can get from them is not the greatest. The Pi Foundation does an excellent job at keeping the cost of their boards down, but that comes with some compromises with audio being one of them. The [audio circuitry](https://hackaday.com/2018/07/13/behind-the-pin-how-the-raspberry-pi-gets-its-audio/) does an OK job, but it's nothing stellar."

Reply:
Make a good job... That's the spanish talking...